Add documentation for tctl's command line arguments for TLS (#125)
temporalio/documentation: docs/tctl.md

```

As you add the bad binary checksum to your namespace, Temporal will not dispatch any decision tasks to the bad binary. So make sure that you have rolled back to a good deployment(or roll out new bits with bug fixes). Otherwise your workflow can't make any progress after auto-reset.

#### Secure connection to Temporal cluster

`tctl` supports optional Transport Level Security (TLS) for secure communication with Temporal, authentication of the server, and authentication of the client (mutual TLS).

`--tls_ca_path=<certificate file path>` command line argument allows to pass a certificate authority (CA) certificate for validating server `tctl` is connecting to.

`--tls_cert_path=<certificate file path>` command line argument allows to pass a certificate for server to validate the client (`tctl`) identity. Requires `--tls_key_path` to also be provided.

> `--tls_key_path=<private key file path>` command line argument allows to pass a private key ofr secure communication with the server. Requires `--tls_key_path` to also be provided.

Reviewer: ```suggestion
`--tls_key_path=<private key file path>` command line argument that passes a private key for secure communication with the server. Requires that `--tls_key_path` is also provided.
```
